docs/server/concepts/all_assignments.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Getting All Assignments Instantaneously
+sidebar_label: Getting All Assignments
+slug: /server/concepts/all-assignments
+---
+
+:::info
+Our SDKs are designed to be invoked in your code at the point where you're serving the experiment treatment or feature. 
+For this reason, we recommend leveraging the standard approach of calling `getExperiment`, `checkGate`, or `getLayer` supplying the user object and the key as needed to determine what to serve. 
+:::
+
+Often times, customers have the desire to capture all assignments (experiment, gates) in a single call on the server for a each user at the start of their session.
+This practice is commonly done as a means to pass assignment information to the client-side a single time, in order to inform the client app of the test groups and features it should apply.
+
+For this use-case, we recommend using the [Client Bootstrapping](/client/concepts/bootstrapping) approach via the `getClientInitializeResponse` server sdk call to generate the full assignments payload (note that the experiment and feature keys within are hashed for privacy) that gets passed down to the client SDK. 
```

Comment:
I dont think id call it the "client bootstrapping approach.

But should add a note on how this method is intented to bootstrap our client sdks (so thats what dictates the format), and therefore it also hashes names by default

Reply:
updated it using this language